libpod/container_inspect.go

```diff
@@ -0,0 +1,70 @@
+package libpod
+
+import (
+	"github.com/projectatomic/libpod/libpod/driver"
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) getContainerInspectData(ctr *Container, size bool, driverData *driver.Data) (*ContainerInspectData, error) {
```

Comment:
Can this be func (ctr *Container) getContainerInspectData(...)? We can get to the runtime as ctr.runtime, so we can drop a parameter that way

Reply:
fixed